src/simulation/dynamics/facetSRPDynamicEffector/facetSRPDynamicEffector.h

```diff
@@ -61,11 +65,11 @@ class FacetSRPDynamicEffector: public SysModel, public DynamicEffector {
     void addArticulatedFacet(Message<HingedRigidBodyMsgPayload> *tmpMsg);
     void ReadMessages();
 
-    uint64_t numFacets = 0;                                                              //!< Total number of spacecraft facets
-    uint64_t numArticulatedFacets = 0;                                                   //!< Number of articulated facets
     ReadFunctor<SpicePlanetStateMsgPayload> sunInMsg;                                    //!< Sun spice ephemeris input message
 
 private:
+    uint64_t numFacets = 0;                                                              //!< Total number of spacecraft facets
```

Comment:
This will break existing code as soon as we release this.  Please keep them as public variables for now.  This gives the user a 1-year grace period to update their code to use the new setter and getter function.  We can't check for the user if they are setting these variables directly, so there is no direct warning.  We'll put a warning in the documentation.  I keep a spreadsheet of depreciated code to remove 1 year later.

Reply:
Done